runner/runner.go

```diff
@@ -245,8 +245,11 @@ func (r *Runner) Shutdown(ctxt context.Context, opts ...client.Option) error {
 	// closed, so send SIGTERM.
```

Comment:
This comment needs to be updated, as the code no longer only handles OSX.

Reply:
I resolved in my latest commit, but let me know if there is anything else needed for this!